Tricky JS usage now producing errors

This construct:

```
       const { name, description, url, home, gmt_offset, timezone_string } = json.data
        return { site_data: { name, description, url, home, gmt_offset, timezone_string } }
```

in lib/wp.js is cute, but is producing errors, now:

```
wp.js
  15:47  error  Identifier 'gmt_offset' is not in camel case       camelcase
  15:59  error  Identifier 'timezone_string' is not in camel case  camelcase

✖ 3 problems (3 errors, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

```

Need to either fix lint config or, better, assign members individually so we lower form js users can understand the code.

--- a/lib/wp.js
+++ b/lib/wp.js
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
 import axios from 'axios'
 import {
   WP_SITE_URL,

